Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -302,7 +333,7 @@ namespace EMotionFX
             case COLUMN_RAGDOLL_LIMIT:
             {
                 const AZStd::shared_ptr<PhysicsSetup>& physicsSetup = m_actor->GetPhysicsSetup();
-                if (physicsSetup)
+                if (!isRootNode && physicsSetup)
```

Comment:
Could we get this check outside of the switch and create a switch for root node and non-root node handling? This might make it safer to maintain in the future.